app/models/application_choice.rb

```diff
@@ -10,7 +10,8 @@ class ApplicationChoice < ApplicationRecord
   audited associated_with: :application_form
 
   scope :for_provider, ->(provider_code) {
-    includes(:course, :provider).where(providers: { code: provider_code })
+    includes(:course, :provider)
```

Comment:
Is this used anywhere outside `GetApplicationChoicesForProvider`? Otherwise it would be better inlined perhaps.

Reply:
Nope. I'll inline it in the service 👍